### DISCUSSION TITLE
Full list of Github Achievements badges?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#18333] [Discussions] [answered]
Saw some of the new Github achievements from github blog.

Does anyone have a full list of all the possible achievements that can be unlocked?

**Comment by melvincwng** [comment, marked as answer]
Hi @roshanMB, you can refer to this link - https://github.com/Schweinepriester/github-profile-achievements for a list of possible achievements in Github.

Please take note, that the list may not be comprehensive at this point in time. There could be more achievements added and updated in that list, in the future.

**Reply by Sanan4li** [reply]
This is a great step by github. I really appreciate this.

**Comment by kai63001** [comment]
nice

**Comment by Akalanka1337** [comment]
Great

**Comment by richaardev** [comment]
Here is a list of all achievements and their variants https://github.com/Schweinepriester/github-profile-achievements

**Reply by dianaledist** [reply]
Thanks!

**Reply by Mark3un** [reply]
Thanks

**Reply by Rayane-T** [reply]
Thanks

**Comment by dianaledist** [comment]
Thank you for sharing the link of the achievements!

**Comment by Benemrxr** [comment]
thank you for sharing

**Comment by Rayane-T** [comment]
Thanks for sharing !

**Comment by knedl1k** [comment]
Thanks :)

**Comment by Shubh2-0** [comment]
Thanks for sharing !

**Comment by jaypee-0** [comment]
Arigato
